README.md

```diff
@@ -8,8 +8,6 @@ A simple role to create a CA to create certificates and deploy them to hosts.
 
 The intended use is to create certificates you can use for connecting clients to main systems. As there are several tools that require keys and certificates with different options set, the role should be able to create them, too. These more specialised keys and certificate must not replace the default ones. See [Contributing](#contributing) below for details. The idea is to run the role once on all hosts in the infrastructure to create all certificates and keys.
 
-The current version is only tested with CentOS 7 and Rocky 8.
-
```

Comment:
Could you change it to the tested versions instead of removing it?

Reply:
I did with the latest commit. It's just that GitHub doesn't show it as part of your thread because I put it 2 lines below the one you commented.